opencog/atomspace/AtomSpace.h

```diff
@@ -358,6 +374,57 @@ class AtomSpace
     }
 
     /**
+     * Gets all the atoms of any type and appends them to the HandleSeq.
+     *
+     * @param appendToHandles the HandleSeq to which to append the handles.
+     *
+     * Example of call to this method:
+     * @code
+     *         std::list<Handle> atoms;
+     *         atomSpace.get_all_atoms(atoms);
+     * @endcode
+     */
+    void get_all_atoms(HandleSeq& appendToHandles) const
+    {
+        // Defer to handles by type for ATOMs and subclasses.
+        get_handles_by_type(appendToHandles, ATOM, true);
```

Comment:
I really really don't like one-line subroutines.   If something takes one line of code to implement, we should not be wrapping it in a wrapper.  It does not save any typing, and it does make it harder to understand the code.